overlord/hookstate/ctlcmd/refresh.go

```diff
@@ -96,6 +102,15 @@ func init() {
 	cmd.hidden = true
 }
 
+func (c *refreshCommand) nonRootExecute(context *hookstate.Context) error {
+	switch {
+	case c.Tracking:
+		return c.printTrackingInfo(context)
+	default:
+		return fmt.Errorf("non-root user only supports --tracking")
```

Comment:
I think we should return a `*ctlcmd.ForbiddenCommandError` here instead of a generic error. The API code checks for that error type to return a 403 instead of a 400, which IMO is the correct behaviour here. This is what already happens if a root-only command is run as non-root